SYCL/GroupAlgorithm/SYCL2020/sort.cpp

```diff
@@ -354,7 +349,7 @@ int main(int argc, char *argv[]) {
     return 0;
   }
 
-  std::vector<int> sizes{1, 2, 64, 256, 1024, 2048, 4096};
+  std::vector<int> sizes{1, 2, 64};
```

Comment:
For this file we have the `items_per_work_item` value. It helps to test the case when one work-item processes multiple buffer elements. So, if work-group size is 4 let's keep following sizes: 1, 12 [case 2) from [the comment](https://github.com/intel/llvm-test-suite/pull/670#issuecomment-1000338927)], 32 [case 3) from [the comment](https://github.com/intel/llvm-test-suite/pull/670#issuecomment-1000338927)]

Reply:
Done.